igor-web/src/main/java/com/netflix/spinnaker/igor/travis/service/TravisService.java

```diff
@@ -324,8 +310,9 @@ public Build getBuild(String repoSlug, int buildNumber) {
             error -> {
               log.warn("An error occurred while fetching new jobs from Travis.", error);
               return Collections.emptyList();
```

Comment:
this seems like a pretty questionable fallback but I don't have skin in the Travis game so I'm not proposing we change anything

Reply:
It's the best fallback. Silent and sneaky!

But yeah, don't change it for now, but I'll try to remember to look into this.